components/dashboard/sections/collectives/CollectiveDetails.tsx

```diff
@@ -339,14 +340,34 @@ const CollectiveDetails = ({
       ) : (
         <React.Fragment>
           <SectionTitle>
-            <LinkCollective
-              collective={collective}
-              className="flex items-center gap-2 font-medium text-slate-700 hover:text-slate-700 hover:underline"
-            >
-              <Avatar collective={collective} radius={48} />
-              {collective.name}
-            </LinkCollective>
+            <Avatar collective={collective} radius={48} />
+            <div>
+              <LinkCollective
+                collective={collective}
+                className="flex items-center gap-2 font-medium text-slate-700 hover:text-slate-700 hover:underline"
+              >
+                {collective.name}
+              </LinkCollective>
+              <div className="text-sm font-normal text-muted-foreground">
+                <FormattedMessage
+                  defaultMessage="{childAccountType} by {parentAccount}"
+                  values={{
+                    childAccountType: (
+                      <Badge size="xs" type="outline">
+                        {formatCollectiveType(intl, collective.type)}
+                      </Badge>
+                    ),
+                    parentAccount: (
+                      <LinkCollective collective={collective.parent} withHoverCard>
+                        {collective.parent.name}
```

Comment:
`parent` may be null here:
![image](https://github.com/opencollective/opencollective-frontend/assets/2119706/a529ff9e-3b13-4e4b-95f0-ade8977c7c64)

Reply:
Oops, good catch!